crates/aide/src/operation.rs

```diff
@@ -200,26 +200,34 @@ pub enum ParamLocation {
 #[tracing::instrument(skip_all)]
 pub fn parameters_from_schema(
     ctx: &mut GenContext,
-    schema: SchemaObject,
+    schema: Schema,
     location: ParamLocation,
 ) -> Vec<Parameter> {
     let schema = ctx.resolve_schema(&schema);
 
     let mut params = Vec::new();
-    if let Some(obj) = &schema.object {
-        for (name, schema) in &obj.properties {
-            let s = schema.clone().into_object();
+
+    if let Some(obj) = schema.as_object() {
+        for (name, schema) in obj {
+            let json_schema: Schema = schema.clone().try_into().expect("Failed to convert schema");
```

Comment:
Same is in the other comment. Here an argument could be made, that we could just `continue` if this code fails. But what would pose a silent failure, which may be even more problematic than just failing and having the user correct their mistakes.